### PR TITLE
[RUN] Implement Vallation, Valiance, Battuta, Inspiration

### PR DESCRIPTION
**Pull request by WinterSolstice8**
Fairly meaty again, started to go too far so cut new features at this line. The result of this PR will help shape the future for the complication that will be Effusion, for sure.

**_I affirm:_**
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [X] that I agree to LandSandBoat's [Limited Contributor License Agreement](https://github.com/LandSandBoat/server/blob/base/.github/CONTRIBUTOR_AGREEMENT.md), as written on this date
- [X] that I have **read the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)**
- [X] that I've _**tested my code and things my code changed**_ since the last commit in the PR, and will test after any later commits

### CODE REVIEW

File: src/map/modifier.h [hunk 1]
```diff
@@ -876,8 +879,8 @@ enum class Mod
     // 261 to 287
     // 888
     // 936
-    // 
-    // SPARE = 1021, and onward
+    //
+    // SPARE = 1023, and onward
```
Comment (Xaver-DaRed) [inline src/map/modifier.h]: 1024

Comment (WinterSolstice8) [reply src/map/modifier.h]: counting is hard :)

**Comment by zach2good** [comment]
Seems fine to me, but since we're targeting a feature branch at the moment it might be a safer idea to PR those additional modifiers to `base`, to save on any possible heartbreak later

**Comment by WinterSolstice8** [comment]
> Seems fine to me, but since we're targeting a feature branch at the moment it might be a safer idea to PR those additional modifiers to `base`, to save on any possible heartbreak later

created #1394 to address this

**Comment by WinterSolstice8** [comment]
Here we go again with rebasing doing things I wasn't expecting by adding so many commits. I need to understand why this keeps happening -- at least I know how to fix this particular issue.

**Comment by WinterSolstice8** [comment]
Looks like some commits went missing from feature/runefencer? As a bonus I figured out how to rebase properly

**Comment by zach2good** [comment]
Yeah, I think I screwed up when I was rebasing :( I can try and fix it after work, or you can grab the missing commits as part of this branch and we'll fix things that way (those commits will still exist/be visible through the previous PRs you've done)

**Review by zach2good — APPROVED** [review]

**Pull request merged**